base/applications/mmc/console.c

```diff
@@ -334,6 +337,16 @@ FrameOnCommand(HWND hwnd,
             PostMessage(hwnd, WM_CLOSE, 0, 0);
             break;
 
+        case IDM_HELP_ABOUT:
+            if (AllocAndLoadString(&lpTitle, hAppInstance, IDS_APPTITLE))
+            {
+                hIcon = LoadIcon(hAppInstance, MAKEINTRESOURCEW(IDI_MAINAPP));
+                ShellAbout(hwnd, lpTitle, NULL, hIcon);
+                DestroyIcon(hIcon);
```

Comment:
MsDoc says not to call it after `LoadIcon()`.
```suggestion
```
Then, you could drop the variable to.

Reply:
oh?
Where does it state that?

Reply:
So I keep it, right ? (looks like we have DestroyIcon in similar places in other ReactOS apps)

Reply:
@SergeGautherie : any documentation or evidence on this ?

Reply:
In https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-destroyicon
> ## Remarks
> It is only necessary to call DestroyIcon for icons and cursors created with the following functions: [CreateIconFromResourceEx](https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-createiconfromresourceex) (if called without the LR_SHARED flag), [CreateIconIndirect](https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-createiconindirect), and [CopyIcon](https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-copyicon). Do not use this function to destroy a shared icon. A shared icon is valid as long as the module from which it was loaded remains in memory. The following functions obtain a shared icon.
    [LoadIcon](https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-loadicona)
    [LoadImage](https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-loadimagea) (if you use the LR_SHARED flag)
......

Reply:
Thanks !
At least I learned something new.
Bad news is that there's a huge cleanup to do in ReactOS to comply with this documentation item.

Reply:
Now fixed

Reply:
> there's a huge cleanup to do in ReactOS

I noticed. I file [CORE-18369](https://jira.reactos.org/browse/CORE-18369) now.

Reply:
Thanks. Good luck 😃